### ISSUE TITLE
 Misbehaving GUI when i through libigl to use imgui

### ISSUE THREAD

**Issue by WhyNot134** [ocornut/imgui#7606]
### Version/Branch of Dear ImGui:

Version 1.90, Branch: master (master/docking/etc.)

### Back-ends:

imgui_impl_glfw.cpp + imgui_impl_opengl3.cpp

### Compiler, OS:

Windows10 + MSVC2019

### Details:

Menu is drawn fine in the libigl EXAMPLE 106 but problem comes when I interact with the window.

- Before clicking anything, hovering over menu items highlights them as I would expect, but if I click on one of them (say, "Show texture", for example), A) nothing happens in the window, B) this is now the only GUI item that highlights when the mouse is hovered over it. I cannot interact with the mesh, rotating, scaling etc with the mouse or keyboard. The GUI is essentially now locked. Clicking on the same GUI item does nothing.
- If, upon the window opening, I first interact with the mesh with a click (ie rotating, translating etc) with the mouse, this is now the only thing I can interact with. I cannot use the GUI menu to select checkboxes etc. The GUI elements do not highlight when I hover over them.
- If, upon first opening, I use the hotkeys for stuff (such as "f" to toggle face-based rendering), the corresponding GUI checkbox changes. Once I click on anything however, the GUI locks again.

Using latest versions of libigl, imgui etc.

### Screenshots/Video:

https://github.com/ocornut/imgui/assets/105153489/7fa7821c-1b93-4478-8e1c-bebd23b8a892

https://github.com/ocornut/imgui/assets/105153489/8ce591ff-a476-484e-a5bb-64dbd7a66fc2

### Minimal, Complete and Verifiable Example code:

```cpp
// Here's some code anyone can copy and paste to reproduce your issue
ImGui::Begin("Example Bug");
MoreCodeToExplainMyIssue();
ImGui::End();
```

**Comment by ocornut** [comment]
I have no idea what igl is nor what the code looks like.

**Comment by GamingMinds-DanielC** [comment]
Since you didn't post any code, I have to assume that the problem occurs with unmodified example sources from the libigl project. I'm not familiar with libigl, but it looks like this issue would be better located in that projects issue section.

**Comment by wenduof** [comment]
It seems like I have the same issue as the details part described above. env: Ubuntu 22.04 + latest libigl + imgui 1.90.8. when I revert imgui version to 1.86, the issue disappears.

**Comment by timelesscat007** [comment]
> It seems like I have the same issue as the details part described above. env: Ubuntu 22.04 + latest libigl + imgui 1.90.8. when I revert imgui version to 1.86, the issue disappears.

Yes! ! ! You are right! when I revert imgui version to 1.86, the issue disappears.Thank you very much for the tip! My env:Windows11 + libigl2.5.0 + imgui 1.86 is SUCCESSFUL! ! !

**Comment by ocornut** [comment]
It is not "successful" to revert to 1.86. It is an unsupported 40+ months old version.
We are open to helping you all but none of you actually posted any relevant imgui-side details nor code, in order for us to help you.

Since you all three are having a similar issue I would suggest it is an issue in libigl. **Please report it there.** Happy to help anyone coming with clear questions.

e.g. I don't understand what a `ImGuiWidget` correspond to in libigl:
https://github.com/libigl/libigl/blob/8866f214a0ca7326971238863c70037d9b2134fa/include/igl/opengl/glfw/imgui/ImGuiPlugin.cpp#L45
https://github.com/libigl/libigl/blob/8866f214a0ca7326971238863c70037d9b2134fa/include/igl/opengl/glfw/imgui/ImGuiWidget.h#L31

It seems like some kind of abstraction and it's possible that the `if(ImGui::GetIO().WantCaptureMouse){ return true; }` logic is faulty in libigl if input of ImGuiPlugin is not propagated to ImGuiWidget. But then I don't understand what it is trying to do.

Please open an issue in libigl repo. If it's proven that there's an issue in Dear ImGui I am happy to investigate.
